### ISSUE TITLE
CLI: marked --help throws

### ISSUE THREAD

**Issue by cancerberoSgx** [markedjs/marked#1587]
**Describe the bug**
Command line interface documentation is non-existent. Running `marked --help` throws an error.

Verified both in macOS and arch linux.
Node.js version: 13.1 and 13.3.

**How to reproduce**

(this also happens when installing locally and run with npx)

```
npm i -g marked
marked --help 
/man/marked.1: No such file or directory
No manual entry for /man/marked.1
marked --version
0.8.0
```

**Expected behavior**

Executing `marked --help` should show minimal command line options or at least a link to the list of CLI options.

**Comment by UziTech** [comment]
nice catch

**Comment by Scrum** [comment]
@UziTech resolved in v0.8.1
